mtp_api/apps/disbursement/serializers.py

```diff
@@ -0,0 +1,40 @@
+from django.db.transaction import atomic
+from rest_framework import serializers
+
+from .models import Disbursement, Recipient
+from .signals import disbursement_created
+
+
+class RecipientSerializer(serializers.ModelSerializer):
```

Comment:
how will we try to re-use existing models?

Reply:
User would get list of recipients filtered by prisoner/prison/whatever (not implemented because it hasn't actually been designed in), then supply chosen id when creating a new disbursement.